pyproject.toml

```diff
@@ -148,6 +148,8 @@ google-auth-oauthlib = { version = "^1.2.0", optional = true }
 google-auth = { version = "^2.25.2", optional = true }
 google-auth-httplib2 = { version = "^0.2.0", optional = true }
 google-api-core = { version = "^2.15.0", optional = true }
+pandas = {version = "^2.1.4", optional = true}
+openpyxl = {version = "^3.1.2", optional = true}
```

Comment:
Do you need this dependency? I don't see it getting used anywhere though.

Reply:
Those are optional for loading Excel files as the Excel Loader depends on pandas and pandas uses openpyxl (or xlsxwriter).